cmd/admin/key.go

```diff
@@ -146,6 +158,37 @@ func (a *admin) spkiHashesFromCertPEM(filename string) ([][]byte, error) {
 	return [][]byte{spkiHash[:]}, nil
 }
 
+func spkiHashFromCSRPEM(filename string, checkSignature bool) ([][]byte, error) {
```

Comment:
Nit: Is there a reason not to follow the pattern of `func (a *admin)` that we're using elsewhere?

Reply:
There's no need for it to be a method - I was considering un-methoding the other input functions here which also don't need to be methods, but in another PR